### DISCUSSION TITLE
Frontmatter shared enabled -> How to use it?

### DISCUSSION THREAD

**Discussion by Boldie** [gethinode/hinode#851] [Help] [answered]
I want to disable the social sharing information for some specific pages. The doc reads like it is possible: https://gethinode.com/docs/configuration/layout/

I tried to disable sharing by adding the following stuff to my frontmatter:
```
---
title: My Title
params:
    sharing:
        enabled: false
---
```

Disabling it in the params.toml in the config directory works as expected, but I want it beeing disabled only for some pages.

```
[sharing]
    enabled = false
```

Any ideas what I have missed?

**Comment by markdumay** [comment, marked as answer]
The [docs](https://gethinode.com/docs/configuration/layout/#social-sharing-configuration) contained an error, which has been corrected now. Add the following setting to your frontmatter to disable social sharing for an individual page (`yaml` syntax):

```yml
---
sharing: false
---

**Reply by Boldie** [reply]
Thanks for your help!
